cmd/mattermost/commands/webhook.go

```diff
@@ -262,6 +270,31 @@ func createOutgoingWebhookCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func deleteWebhookCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("WebhookID is not specified")
+	}
+
+	webhookId := args[0]
+	errIncomingWebhook := app.DeleteIncomingWebhook(webhookId)
+	errOutgoingWebhook := app.DeleteOutgoingWebhook(webhookId)
```

Comment:
@jespino It's not specified in the ticket but I wonder if should we separate deleting of webhook as incoming and outgoing, similar to how we're doing in creating and modifying?

Reply:
The creation and the modification are specific for each webhook type, they have different fields, but the deletion can be in the same command. Actually I defined it this way in the task description. We can revisit this decision later and change it in another PR.